### ISSUE TITLE
Router: Query param change fails to make it to _onUrlParams()

### ISSUE THREAD

**Issue by frank-weindel** [rdkcentral/Lightning-SDK#274]
Starting with already being on a page with the route:
```
#front-door
```

Manually add a URL param to the hash in the browser to look like so:
```
#front-door?myParam=myValue
```

**Expected:** _onUrlParams' `param` is:
```
{
  Symbol(queryParams): {
    myParam: "myValue"
  }
  Symbol(store): undefined
}
```

**Actual:** _onUrlParams' `param` is:
```
{
  Symbol(store): undefined
}
```

**Comment by erikhaandrikman** [comment]
This was added to capture url parameters when a `bootComponent` is configured. When it's configured the Router will navigate to `$` force it to load as any other configured route.

https://github.com/rdkcentral/Lightning-SDK/blob/93c749509bb9a5db5e2888b8eed7d76ab3ac72ec/src/Router/index.js#L85
https://github.com/rdkcentral/Lightning-SDK/blob/93c749509bb9a5db5e2888b8eed7d76ab3ac72ec/src/Router/index.js#L132

Depending on the moment you call `getQueryStringParams()` ; `getActiveHash()` can return `undefined` or `$` ( in case of calling it in `root` function ) to still be able to read the correct parameters, it uses the `resumeHash` instead of the active hash: https://github.com/rdkcentral/Lightning-SDK/blob/93c749509bb9a5db5e2888b8eed7d76ab3ac72ec/src/Router/utils/helpers.js#L114

The problem seems to be in the condition; it should check if hash eq to `$` or `undefined` AND `resumeHash` has a value

https://github.com/rdkcentral/Lightning-SDK/blob/93c749509bb9a5db5e2888b8eed7d76ab3ac72ec/src/Router/utils/helpers.js#L116

**Comment by erikhaandrikman** [comment]
This fix is available in release `4.8.0`

**Comment by frank-weindel** [comment]
Thanks @erikhaandrikman
